src/views/blocks/parts/organizer.php

```diff
@@ -44,7 +44,9 @@
 			}
 
 			?>
-			<dt style="display:none;"><?php // This element is just to make sure we have a valid HTML ?></dt>
+			<dt class="tribe-common-a11y-visual-hide">
```

Comment:
Since this is visually hidden, I'd look into if some text here would be useful for screen readers (for all of these)

Reply:
@Camwyn I've added some `aria-label`s for screen readers.